tests/ExporterTest.php

```diff
@@ -187,16 +187,46 @@ public function testGetProductsException()
         $exporterMock->getProducts();
     }
 
+    public function providerCreateProductItem()
+    {
+        return array(
```

Comment:
Use the `[ ]` notation instead of `array`.

Reply:
Should I really switch to [ ]  as it would make code standart inconsistent ? I think it was mentioned in one of the previous pull request by you